metadata-io/src/test/java/com/linkedin/metadata/search/elasticsearch/fixtures/SampleDataFixtureTests.java

```diff
@@ -119,8 +119,8 @@ public void testSearchFieldConfig() throws IOException {
             for (SearchableFieldSpec fieldSpec : entitySpec.getSearchableFieldSpecs()) {
                 SearchFieldConfig test = SearchFieldConfig.detectSubFieldType(fieldSpec);
 
-                if (!test.getFieldName().contains(".")) {
-                    Map<String, Object> actual = mappings.get(test.getFieldName());
+                if (!test.fieldName().contains(".")) {
```

Comment:
A bit curious on these changes, guess this is coming from the "fluent" annotation. Is this just a preference over having the "get" in the name or is it relevant for the deserialization?

Reply:
Without the fluent setting is creates methods like `isHasMyBoolean()` and I don't like `isHasKeywordField()` preferring instead `hasKeywordField()`